lib/cpp/src/thrift/TApplicationException.h

```diff
@@ -21,6 +21,7 @@
 #define _THRIFT_TAPPLICATIONEXCEPTION_H_ 1
 
 #include <thrift/Thrift.h>
+#include <stdint.h>
```

Comment:
Oh... it sounds pretty much caused by THRIFT-2835... I'll look into it within a few days or so too.